tests/test_pydough_to_sql.py

```diff
@@ -25,42 +25,41 @@
 
 
 @pytest.mark.parametrize(
-    "pydough_code, expected_sql",
-    # Note: All of these tests are for simple code because the
-    # exact SQL generation for inner expressions is currently
-    # non-deterministic.
+    "pydough_code, test_name",
     [
         pytest.param(
             simple_scan,
-            "SELECT o_orderkey AS key FROM tpch.ORDERS",
+            "simple_scan",
             id="simple_scan",
         ),
         pytest.param(
             simple_filter,
-            "SELECT o_orderkey, o_totalprice FROM (SELECT o_orderkey AS o_orderkey, o_totalprice AS o_totalprice FROM tpch.ORDERS) WHERE o_totalprice < 1000.0",
+            "simple_filter",
             id="simple_filter",
         ),
         pytest.param(
             rank_a,
-            "SELECT ROW_NUMBER() OVER (ORDER BY acctbal DESC NULLS FIRST) AS rank FROM (SELECT c_acctbal AS acctbal FROM tpch.CUSTOMER)",
+            "rank_a",
             id="rank_a",
         ),
         pytest.param(
             rank_b,
-            " SELECT RANK() OVER (ORDER BY order_priority NULLS LAST) AS rank FROM (SELECT o_orderpriority AS order_priority FROM tpch.ORDERS)",
+            "rank_b)",
```

Comment:
Is the parenthesis deliberate? I see that `rank_b).sql` file has been generated.